README.md

```diff
@@ -353,6 +353,8 @@ The tool running the Pebble server can make use of this, for example, under Kube
 
 Ready implies alive, and not-alive implies not-ready. If you've configured an "alive" check but no "ready" check, and the "alive" check is unhealthy, `/v1/health?level=ready` will report unhealthy as well, and the Kubernetes readiness probe will act on that.
 
+On the other hand, not-ready does not imply not-alive: if you've configured a "ready" check but no "alive" check, and the "ready" check is unhealthy, `/v1/health?level=alive` will still report healthy.
```

Comment:
Tangential: given that this needed to be clarified, how about this:

I'm assuming that it's possible to specify both ready and health checks on the same service.

What if their results are in conflict? That is raw results are `ready: true; alive: false`?

Reply:
Or is it perhaps that "ready implies alive, if the latter is not specified"?

Reply:
I've (hopefully) clarified slightly in https://github.com/canonical/pebble/pull/442/commits/c1fe1d346d2b302b403508241f909f3d3d7701a1

If they're both specified, they're not in conflict, it's just that there's no fallback: /v1/health?level=ready gives you the `ready` checks, and /v1/health?level=alive gives you the `alive` checks directly.